ci/prow/plugins.yaml

```diff
@@ -33,6 +33,26 @@ repo_milestone:
     maintainers_id: 3012514
     maintainers_team: knative-milestone-maintainers
 
+project_config:
+  project_org_configs:
+    knative:
+      org_maintainers_team_id: 2652083 # knative-admin
+      # TODO(Fredy-Z): Enable this plugin for other repos when needed.
+      project_repo_configs:
+        eventing:
+          repo_maintainers_team_id: 3058996 # eventing-admins
```

Comment:
For now can we use `knative-milestone-maintainers` here?

Once we verify the plugin works, I'd like to create a new team `eventing-project-maintainers` but that can be a later PR.

Reply:
Done.